pages/_document.tsx

```diff
@@ -1,4 +1,4 @@
-import { ServerStyleSheets } from '@material-ui/core/styles';
+import { ServerStyleSheets } from '@mui/styles';
```

Comment:
What does this affect?

Reply:
https://github.com/mui/material-ui/issues/17850